src/reducers/import.js

```diff
@@ -9,6 +15,15 @@ import {
   IMPORT_IMPORT_SUCCESS
 } from '@/actions/import'
 
+const enabled = (state = false, action) => {
+  switch (action.type) {
+    case SET_IMPORTS_ENABLED:
+      return !!action.enabled
+    default:
+      return state
+  }
+}
```

Comment:
redux is an old school way to handle state in an app. We should avoid using it, specially for small data.